website/blog/2022-11-22-move-spreadsheets-to-your-dwh.md

```diff
@@ -0,0 +1,191 @@
+
+Because of spreadsheets’ importance as the glue between many business processes, there are different tools to load them into your data warehouse and each one has its own pros and cons, depending on your specific use case.
+
+<!--truncate-->
+
+In general, there are a few questions to ask yourself about your data before choosing one of these tools:
+
+- Who at your company will be loading the data?
+- Does it have a consistent format?
+- How frequently will it change?
+- How big is the dataset?
+- Do changes need to be tracked?
+- Where are the files coming from?
+
+Let’s have a look at some of the offerings to help you get your spreadsheets into your data warehouse.
+
+## dbt seeds
+
+If you’re using [dbt](https://www.getdbt.com/) to manage your data transformations, it comes with an inbuilt csv loader ([seeds](https://docs.getdbt.com/docs/building-a-dbt-project/seeds)) to populate your data warehouse with any files you put inside of your project’s `seeds` folder. It will automatically infer data types from your file’s contents, but you can always override it by [providing explicit instructions in your dbt_project.yml](https://docs.getdbt.com/reference/resource-configs/column_types) file.
+
+However, since dbt creates these tables by inserting rows one at a time, it doesn’t perform well at scale (there’s no hard limit but syncing a hundred rows should be fine, as opposed to a few thousand). [The dbt docs](https://docs.getdbt.com/docs/building-a-dbt-project/seeds#faqs) suggest using seeds for “files that contain business-specific logic, for example, a list of country codes or user IDs of employees.”
+
+A big benefit of using seeds is that your file will be checked into source control, allowing you to easily see when the file was updated and retrieve deleted data if necessary.
+
+#### Good fit for:
+
+- Small files such as mapping employee identifiers to employees
+- Infrequently modified files such as mapping country codes to country names
+- Data that would benefit from source control
+- Programmatic control of data types
+
+#### Not a good fit for:
+
+- Files greater than 1MB in size
+- Files that need regular updates
+
+## ETL tools
+
+An obvious choice if you have data to load into your warehouse would be your existing [ETL tool](https://www.getdbt.com/analytics-engineering/etl-tools-a-love-letter/) such as Fivetran or Stitch, which we'll dive into in this section. Below is a summary table highlighting the core benefits and drawbacks of certain ETL tooling options for getting spreadsheet data in your data warehouse.
+
+### Summary table
+
+| Option/connector | Good for frequently updated data | Good for change tracking | Configurable data types |
+| --- | --- | --- | --- |
+| dbt seeds | ❌ | ✅ | ✅ |
+| Fivetran Browser Upload | ✅ | ❌ | ✅ |
+| Fivetran Google Sheets connector | ✅ | ❌ | ❌ |
+| Fivetran Google Drive connector | ❌ | ❌ | ❌ |
+| Stitch Google Sheets integration | ✅ | ❌ | ❌ |
+| Airbyte Google Sheets connector | ✅ | ❌ | ❌ |
```

Comment:
I know you thought extra columns might make it clunky, but I think these are important 😅 
```suggestion
| Option/connector | Data updatable after load |  Configurable data types | Multiple tables per schema | Good for large datasets |
| --- | --- | --- | --- | --- |
| dbt seeds | ✅ | ✅ | ✅ | ❌ |
| Fivetran Browser Upload | ✅ | ✅ | ✅ | ✅ |
| Fivetran Google Sheets connector | ✅ | ❌ | ❌ | ✅ |
| Fivetran Google Drive connector | ❌ | ❌ | ✅ | ✅ |
| Stitch Google Sheets integration | ✅ | ❌ | ❌ | ✅ |
| Airbyte Google Sheets connector | ✅ | ❌ | ❌ | ✅ |
```

It now looks like 
![image](https://user-images.githubusercontent.com/7335046/203656662-9ed79acb-11b2-4720-b5ae-4ca9cac908d0.png)